packages/console/src/utilities/validator.ts

```diff
@@ -1,6 +1,6 @@
-export const uriValidator = (verifyBlank = true) => {
+export const uriValidator = ({ allowBlank = true }) => {
   return (value: string) => {
-    if (!verifyBlank && value.trim().length === 0) {
+    if (!allowBlank && value.trim().length === 0) {
       return true;
     }
```

Comment:
seems that if `allowBlank` is false, we should return false here?

Reply:
ditto